src/ONYXKEYS.ts

```diff
@@ -527,6 +527,12 @@ const ONYXKEYS = {
     /** Set this gets redirected from global reimbursements flow */
     IS_COMING_FROM_GLOBAL_REIMBURSEMENTS_FLOW: 'isComingFromGlobalReimbursementsFlow',
 
+    /** Timestamp of the last login on iOS */
+    NVP_LAST_ECASH_IOS_LOGIN: 'nvp_lastECashIOSLogin',
+
+    /** Timestamp of the last login on Android */
+    NVP_LAST_ECASH_ANDROID_LOGIN: 'nvp_lastECashAndroidLogin',
```

Comment:
I think we should also add the `lastAndroidLogin` and `lastiPhoneLogin` nvps here. Since those are also for our mobile apps.